<ui version="4.0" >
 <class>ViewerPageBase</class>
 <widget class="QWidget" name="ViewerPageBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="fCategoryLabel" >
       <property name="text" >
        <string>Category:</string>
       </property>
       <property name="buddy" >
        <cstring>fCategories</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KComboBox" name="fCategories" />
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QListWidget" name="fRecordList" />
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="fRecordInfoLabel" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text" >
        <string>Record Info:</string>
       </property>
       <property name="buddy" >
        <cstring>fRecordInfo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KTextEdit" name="fRecordInfo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
